decred_wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -101,6 +109,13 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
     
     func seedWordEntered(for wordIndex: Int, seedWord: String, moveToNextField: Bool) {
         self.userEnteredSeedWords[wordIndex] = seedWord
+        self.lblEnterAllSeeds.isHidden = true
```

Comment:
Enclose lines 112 to 118 in an if statement. There's especially no need to call `UIView.animate` if nothing is being changed on the UI. Should first check that `self.lblEnterAllSeeds.isHidden == false`